Sources/RequestsAPI.swift

```diff
@@ -77,9 +77,10 @@ open class RequestsAPI: NSObject {
      A JSON representation of the HTTP response body.
      */
     open class func getJSON(_ url: URL) -> [String: Any]? {
-        let data = self.get(url)
-
-        if let json = try? JSONSerialization.jsonObject(with: data!, options: []) as? [String: Any] {
```

Comment:
Are there any other unsafe code like that?

Reply:
Found one, Fixing it for another potential bug https://github.com/okta/okta-ios-jwt/blob/master/Sources/OktaJWT.swift#L255